### PR TITLE
Modify sort in category

### PR DESCRIPTION
**Pull request by Mina-Wael**

### CODE REVIEW

**Review by AhmedAbdallah125 — APPROVED** [review]

File: app/src/main/java/com/iti/android/team1/ebuy/ui/category/view/CategoryFragment.kt [hunk 1]
```diff
@@ -1,6 +1,7 @@
 package com.iti.android.team1.ebuy.ui.category.view
 
 import android.os.Bundle
+import android.util.Log
```
Comment (MohabElziny) [inline app/src/main/java/com/iti/android/team1/ebuy/ui/category/view/CategoryFragment.kt:4]: remove the log from the code

File: app/src/main/java/com/iti/android/team1/ebuy/ui/category/viewmodel/CategoryViewModel.kt [hunk 2]
```diff
@@ -1,5 +1,6 @@
 package com.iti.android.team1.ebuy.ui.category.viewmodel
 
+import android.util.Log
```
Comment (MohabElziny) [inline app/src/main/java/com/iti/android/team1/ebuy/ui/category/viewmodel/CategoryViewModel.kt:3]: remove the log

**Review by MohabElziny — CHANGES_REQUESTED** [review]

File: app/src/main/java/com/iti/android/team1/ebuy/ui/category/view/CategoryFragment.kt (hunk 1 above)
Comment (Mina-Wael) [reply app/src/main/java/com/iti/android/team1/ebuy/ui/category/view/CategoryFragment.kt]: done

File: app/src/main/java/com/iti/android/team1/ebuy/ui/category/viewmodel/CategoryViewModel.kt (hunk 2 above)
Comment (Mina-Wael) [reply app/src/main/java/com/iti/android/team1/ebuy/ui/category/viewmodel/CategoryViewModel.kt:3]: done

**Review by MohabElziny — APPROVED** [review]

**Pull request merged**